react/ecmascript/02 - promises/index.js

```diff
@@ -0,0 +1,49 @@
+// Hint: use setInterval, create a new Promise and measure time with Date.now()
+
+export function delay(time) {
+
+var start = Date.now();
+
+return new Promise(
+    function (resolve, reject) {
+      if(time >= 400 && time <= 600) {
+        setTimeout(function(){
```

Comment:
here you can write `setTimeout(() => resolve(Date.now() - start), time);` Use arrow function here